app/resources/v1/pacbio/library_resource.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
+require './app/resources/v1/shared/run_suitability'
```

Comment:
Why do we need to require this? Is there an issue with loading.

Reply:
There was, but looking at it now, I think its just because I haven't also name-spaced the module itself under shared.